src/basic/syscalls-alpha.txt

```diff
@@ -401,7 +404,7 @@ pwritev	491
 pwritev2	521
 query_module	347
 quotactl	148
-quotactl_path	553
+quotactl_path
```

Comment:
Why is `quotactl_path` getting dropped? Did it get reverted?

Reply:
yes, it did. https://lwn.net/Articles/859679/

Reply:
Oh, https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=5b9fedb31e476693. It would be good mention this in the commit message.